### PR TITLE
2020 12 update vtkm

### PR DESCRIPTION
**Pull request by mclarsen**

### CODE REVIEW

**Comment by mclarsen** [comment]
@cyrush I think I have everything in place inside of ascent as well. I haven't tested on the GPU, but i'll get to that.

**Comment by cyrush** [comment]
looks good, I'll check this out build the new CI containers and change the CI to use them.

File: src/tests/vtkh/CMakeLists.txt [hunk 1]
```diff
@@ -84,13 +84,9 @@ set(BASIC_TESTS t_vtk-h_smoke
...
                 t_vtk-h_slice
                 t_vtk-h_volume_renderer
+                ${t_vtk-h_contour_tree_}
                 )
 
-# this was already disabled, to enable uncomment:
-# if(ENABLE_FILTER_CONTOUR_TREE)
-#     list(APPEND BASIC_TESTS ${t_vtk-h_contour_tree_})
-# endif()
-
```
Comment (cyrush) [inline src/tests/vtkh/CMakeLists.txt:93]: we need this logic back

File: src/tests/vtkh/CMakeLists.txt [hunk 2]
```diff
@@ -100,18 +96,13 @@ set(MPI_TESTS t_vtk-h_smoke_par
...
               t_vtk-h_sampling_par
               t_vtk-h_volume_renderer_par
+              ${t_vtk-h_contour_tree_par_}
               )
 
-# this was already disabled, to enable uncomment:
-# if(ENABLE_FILTER_CONTOUR_TREE)
-#   list(APPEND BASIC_TESTS ${t_vtk-h_contour_tree_par_})
-# endif()
-
```
Comment (cyrush) [inline src/tests/vtkh/CMakeLists.txt:114]: also need this logic back

File: src/vtkh/filters/CMakeLists.txt [hunk 3]
```diff
@@ -76,11 +70,6 @@ set(vtkh_filters_sources
   communication/MemStream.cpp
   )
 
-if(ENABLE_FILTER_CONTOUR_TREE)
-  list(APPEND vtkh_filters_headers ContourTree.cpp)
-endif()
-
-
```
Comment (cyrush) [inline src/vtkh/filters/CMakeLists.txt:83]: we need to add this back

File: src/vtkh/filters/MarchingCubes.cpp [hunk 4]
```diff
@@ -1,9 +1,6 @@
 #include <vtkh/filters/MarchingCubes.hpp>
 #include <vtkh/Error.hpp>
-
-#ifdef VTK_H_ENABLE_FILTER_CONTOUR_TREE
 #include <vtkh/filters/ContourTree.hpp>
-#endif
```
Comment (cyrush) [inline src/vtkh/filters/MarchingCubes.cpp]: need this back

**Pull request merged**